lib/pbench/server/api/resources/users_api.py

```diff
@@ -298,8 +299,22 @@ def __init__(self, logger, auth):
         self.logger = logger
         self.auth = auth
 
+    def operation_permitted(self, target_username: str) -> User:
```

Comment:
Something like `user_if_permitted` would provide a better description of what it does.